gh-prs

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+repo="dlvhdr/gh-prs"
+tag="v0.1.1"
```

Comment:
How are we going to maintain this value?
If I understand correctly, the piece of code that uses it below is for when a user clones this repo and doesn't install the extension through the `gh` cli, right?
And if that's the case, we download the right exe from the release.

In that case, can't we download the latest release?
I checked, and by not providing a tag - `gh release download` will fetch the latest version.
So I think it's safe to remove

Reply:
I test it by install it like this

```bash
gh extension install abdfnx/gh-prs
```

and it's works correctly

Reply:
Sorry, I misspelled the tag